cmd/openshift-install/create.go

```diff
@@ -320,8 +370,8 @@ func waitForBootstrapComplete(ctx context.Context, config *rest.Config) (err err
 // waitForBootstrapConfigMap watches the configmaps in the kube-system namespace
 // and waits for the bootstrap configmap to report that bootstrapping has
 // completed.
-func waitForBootstrapConfigMap(ctx context.Context, client *kubernetes.Clientset) error {
-	timeout := 30 * time.Minute
+func waitForBootstrapConfigMap(ctx context.Context, client *kubernetes.Clientset) *clusterCreateError {
+	timeout := 1 * time.Minute
```

Comment:
Oops. Left this one in from your testing.

Reply:
Ah damn sorry my bad

Reply:
No worries. It happens to **everyone**.